scripts/evaluation/rubrics/quality_rubrics.py

```diff
@@ -0,0 +1,54 @@
+"""Definition of grading rubrics for dialogue quality evaluation."""
+
+from enum import Enum
+
+
+class QualityRubrics(Enum):
+    REC_RELEVANCE = "Recommendation Relevance\n5: The agent consistently "
```

Comment:
You could write it as "\n".join(["...", ]) for a bit better readability (while still breaking long lines)